src/modelfree/score_agent.py

```diff
@@ -18,6 +18,7 @@
 from modelfree.common.utils import TrajectoryRecorder, VideoWrapper, make_env, simulate
 from modelfree.envs.gym_compete import GymCompeteToOurs, game_outcome
 from modelfree.envs.observation_masking import make_mask_agent_wrappers
+from modelfree.training.shaping_wrappers import NoisyAgentWrapper
```

Comment:
Only `modelfree.train` should depend on `modelfree.training`. Let's move `NoisyAgentWrapper` somewhere into `common`. Probably `common.utils`, although that's becoming a bit of a kitchen sink. `utils.py` already has things for `DummyModel`, etc -- so we could maybe factor all those out into a new module e.g. `common.policies`? And leave things like `VideoWrapper`, `TrajectoryRecorder`, etc.

Reply:
Would the idea be to keep the NoisyAgentWrapper in the file with the policies, or with the wrappers? It seems to conceptually fit more with the wrappers, but inherits from DummyModel

Reply:
Tricky to communicate so I've moved stuff around to try and address this. Please take a look and let me know if you spot any issues.